cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -35,7 +35,6 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
-
```

Comment:
one minor nit: we should not remove this whitespace line.

Reply:
Done